source/installation/index.html

```diff
@@ -98,7 +117,7 @@ <h2>Windows</h2>
                 <div>
                     <b>Windows</b>
                     <ul>
-                        <li>Home Assistant Operating System</li>
+                        <li>Home Assistant Operating System (using Virtual Machine)</li>
```

Comment:
This makes this item a bit long (especially on mobile). I think this can be emphasized on the page where this leads to. I would simply shorten to VM in these items?

```suggestion
                        <li>Home Assistant Operating System (VM)</li>
```